tests/integration/test_data_cli.py

```diff
@@ -701,9 +702,31 @@ def test_search_get_id_not_found(invoke, search_id):
     assert 'Error: {"message": "Error message"}\n' == result.output
 
 
+@respx.mock
+def test_search_delete(invoke, search_id, search_result):
+    delete_url = f'{TEST_SEARCHES_URL}/{search_id}'
+    mock_resp = httpx.Response(HTTPStatus.NO_CONTENT, json=search_result)
+    respx.delete(delete_url).return_value = mock_resp
+
+    result = invoke(['search-delete', search_id])
+
+    assert not result.exception
+
+
+@respx.mock
+def test_search_delete_invalid_id(invoke, search_id, search_result):
+    delete_url = f'{TEST_SEARCHES_URL}/{search_id}'
+    mock_resp = httpx.Response(404, json=search_result)
+    respx.delete(delete_url).return_value = mock_resp
+
+    result = invoke(['search-delete', search_id])
+
+    assert result.exception
+    assert result.exit_code == 1
```

Comment:
Hrm... Ideally, with an invalid parameter (search_id, here), we'd return code of 2 ([ref](https://github.com/planetlabs/planet-client-python/blob/main/DESIGN.md#cli-return-codes)). But we aren't pre-validating the search-id before we send it to the API. Maybe just change the name of the test to say the search_id is nonexistant instead of invalid? I *think* the existence is really what we're checking here, no?

Reply:
I think you're right